pkgs/tools/misc/nncp/default.nix

```
@@ -0,0 +1,46 @@
+{ stdenv, go, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "nncp";
+  version = "5.3.3";
+
+  buildInputs = [ go ];
+
+  preConfigure = ''
+    export GOCACHE=$PWD/.cache
+  '';
+
+  makeFlags = [
+    "PREFIX=$$out"
```

```suggestion
    "PREFIX=${placeholder "out"}"
```

BTW have you tried without it? From AFAIK this flag is set by default with the default `builder`.

---

Just tried it and it did not work without.

---

How did it fail? See https://nixos.org/nix/manual/#builtin-placeholder

---

It works with placeholder. When removing the PREFIX line entirely, it fails with

`mkdir: cannot create directory '/usr': Permission denied`